### DISCUSSION TITLE
Error for sharepoint file sync - critical or momentary?

### DISCUSSION THREAD

**Discussion by johnsidoli** [abraunegg/onedrive#1801] [General]
Hi, been using the client for a week or so and very happy, it works well. But I am also currently trying to watch the notifications a lot to feel I can"trust" it works etc and noticed this weird error fly by... the file itself seems to have synched in the end... so should I be concerned? i.e. Is it a critical issue or just something that momentarily went wrong? Anything to be wary of?

n.b. this was a sharepoint file, as the error speculates.

```
2022-Jan-15 21:52:05.9484552	Downloading file Book1.xlsx ... done.
2022-Jan-15 21:52:18.37675	ERROR: File download size mis-match. Increase logging verbosity to determine why.
2022-Jan-15 21:52:18.3769054	ERROR: File download hash mis-match. Increase logging verbosity to determine why.
2022-Jan-15 21:52:18.3770555	INFO: It is most likely that a SharePoint OneDrive API issue is the root cause. Add --disable-download-validation to work around this issue but downloaded data integrity cannot be guaranteed.
2022-Jan-15 21:52:18.3782379	
ERROR: The local file system returned an error with the following message:
2022-Jan-15 21:52:18.3783696	  Error Message:    Book1.xlsx: No such file or directory
2022-Jan-15 21:52:18.3784674	  Calling Function: applyChangedItem()

2022-Jan-15 21:52:24.3894806	Downloading file Book1.xlsx ... done.

```

**Comment by abraunegg** [comment]
@johnsidoli 
This is nothing to be concerned about.

Microsoft SharePoint does odd things, including modifying your files post upload (so what you uploaded is not what you thought you uploaded). In some circumstances, the SharePoint site also says that the file size to download is X but in reality Y is what is downloaded - hence this error message / warning.

**Reply by johnsidoli** [reply]
Thanks @abraunegg :)
